apis/database/v1beta1/dbsubnetgroup_types.go

```diff
@@ -82,6 +88,7 @@ type DBSubnetGroupParameters struct {
 	DBSubnetGroupDescription string `json:"description"`
 
 	// The name for the DB subnet group. This value is stored as a lowercase string.
+	// +immutable
 	DBSubnetGroupName string `json:"groupName"`
```

Comment:
We use `meta.GetExternalName(cr)` when we need the name, so, this field should be removed now.